tools/azure_controller.sh

```diff
@@ -24,8 +24,6 @@ echo $MAVEN_OPTS
 mvn -version
 echo "Commit: $(git rev-parse HEAD)"
 
-
-
```

Comment:
unrelated

Reply:
You are right. I need to read the guidelines more closely.